tests/ballerina-unit-test/src/test/resources/test-src/types/xml/xml-literals-negative.bal

```diff
@@ -22,15 +22,15 @@ function defineEmptyNamespaceInline() {
 
 function testExpressionAsElementName() {
     int v1 = 11;
-    // xml x1 = xml `<ns0:{{v1}}>hello</ns0:{{v1}}>`;
+    // xml x1 = xml `<ns0:${v1}>hello</ns0:${v1}>`;
```

Comment:
Isn't this also invalid now? Shall we remove this commented line and also the TODO in the `XMLLiteralTest` if it is no longer valid?